docs/02-install.md

```diff
@@ -144,6 +146,66 @@ file in your installation artifacts directory and simply re-export it:
 
     export KUBECONFIG=/path/to/something/auth/kubeconfig
 
+# Advnaced Installations
+While the OpenShift 4 installer's purpose in life is to streamline operations
+in order to guarantee success, there are a few options that you can adjust by
+using a configuration file. Namely, you can change the default number of
+instances, and you can change both the master and initial worker EC2 instance
```

Comment:
Should add that changing the disk configuration is likely something most people will want in order to provide etcd more iops.

Reply:
How can you change the disk config in the install-config.yaml?

Reply:
Ex:

```
machines:
- name: master
  platform:
    aws:
      type: m5.large
      rootVolume:
        iops: 700
        size: 220
        type: io1
```

Reply:
@akrzos yeah i went digging and found:
https://godoc.org/github.com/openshift/installer/pkg/types/aws#EC2RootVolume 

Will tweak.

Question though -- if we need more iops for etcd, why isn't the default to use more iops? if you only sometimes need more iops, i assume this would need to go in some kind of scaling guide?

Reply:
> Question though -- if we need more iops for etcd, why isn't the default to use more iops? if you only sometimes need more iops...

This is why.  For example, etcd is going to care more about iops than, say, an image registry and other use-cases that can tolerate more latency.

> ... i assume this would need to go in some kind of scaling guide?

A scaling guide could talk about when to use which values.  This document should mention that the values are tunable, but does not need to motivate its choices.

Reply:
OK. I did add wording for etcd so I think we're good.